app/views/admin/collections/index.html.erb

```diff
@@ -1,7 +1,9 @@
 <h1>Collections</h1>
-<p>
-  <%= link_to 'New Collection', new_admin_collection_path, class: "btn btn-primary " %>
-</p>
+<% if can?(:create, Kithe::Model) %>
+  <p>
+    <%= link_to 'New Collection', new_admin_collection_path, class: "btn btn-primary " %>
+  </p>
+<% end %>
```

Comment:
We talked about providing a disabled version of the button somehow, instead of just making it not show up. 

To make it less confusing to people who don't have permissions, and might otherwise wonder -- do I not have permissions, or am I just not finding where the button is?  I guess that applies up in admin_menu_helper too, and other files. 

What do you think?  If you think this is good enough, we could try it.

Reply:
Surprisingly easy to make both changes, and I do in fact like it better. Will do likewise on the other PRs.